tasks/omnibus.py

```diff
@@ -379,3 +379,26 @@ def manifest(
         omnibus_s3_cache=False,
         log_level=log_level,
     )
+
+@task
+def rpath_edit(ctx, install_path, target_rpath_dd_folder, macos=False):
```

Comment:
I'm not a fan of having `macos` as a boolean option here, I'd rather have a `os` or `platform` parameter (that may default to `"linux"` or whatever makes most sense).